### PR TITLE
docs(no-effect-decorator-and-creator): add docs

### PR DESCRIPTION
**Pull request by Armenvardanyan95**
Created the rule doc

### CODE REVIEW

File: docs/rules/no-effect-decorator-and-creator.md [hunk 1]
```diff
@@ -1 +1,39 @@
 # no-effect-decorator-and-creator
+
+> This rule disallows using the `@Effect` decorator and the `createEffect` function simultaneously
+
+## Rule Details
+
+There are two ways we can register an Effect in NgRx. One is using the `@Effect` decorator, the other is with the help of the `createEffect` function. Using both simultaneously will result in the Effect being registered twice, and the side-effect will be performed twice every time the corresponding action is dispatched.
```
Comment (timdeschryver) [inline docs/rules/no-effect-decorator-and-creator.md:7]:
```suggestion
There are two ways we can register an Effect in NgRx. One is using the `@Effect` decorator (this is currently deprecated), the other is with the help of the `createEffect` function. Using both simultaneously will result in the Effect being registered twice, and the side-effect will be performed twice every time the corresponding action is dispatched.
```

File: docs/rules/no-effect-decorator-and-creator.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+Examples of **correct** code for this rule:
+
+```ts
+export class Effects {
+
+  loadData$ = createEffect(() => this.actions.pipe(
+    ofType(loadData),
+    // performing the side effect
+  ));
```
Comment (timdeschryver) [inline docs/rules/no-effect-decorator-and-creator.md:33]:
```suggestion
  loadData$ = createEffect(() => {
    return this.actions.pipe(
      ofType(loadData),
      // performing the side effect
    ))
  };
```

File: docs/rules/no-effect-decorator-and-creator.md (hunk 1 above)
Comment (Armenvardanyan95) [reply docs/rules/no-effect-decorator-and-creator.md:7]: Very nice! I thought about adding a hint that using `@Effect` is not good either, but somehow forgot

File: docs/rules/no-effect-decorator-and-creator.md (hunk 2 above)
Comment (Armenvardanyan95) [reply docs/rules/no-effect-decorator-and-creator.md:33]: I keep writing "good" examples that violate other rules 😁

File: docs/rules/no-effect-decorator-and-creator.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+## Rule Details
+
+There are two ways we can register an Effect in NgRx. One is using the `@Effect` decorator (this is currently deprecated), the other is with the help of the `createEffect` function. Using both simultaneously will result in the Effect being registered twice, and the side-effect will be performed twice every time the corresponding action is dispatched.
+
+Examples of **incorrect** code for this rule:
+
+```ts
+export class Effects {
+
+  @Effect() loadData$ = createEffect(() => this.actions.pipe(
```
Comment (timdeschryver) [inline docs/rules/no-effect-decorator-and-creator.md:14]:
```suggestion
  @Effect() loadData$ = createEffect(() => this.actions$.pipe(
```

File: docs/rules/no-effect-decorator-and-creator.md [hunk 4]
```diff
@@ -1 +1,41 @@
...
+}
+```
+
+Examples of **correct** code for this rule:
+
+```ts
+export class Effects {
+
+  loadData$ = createEffect(() => {
+    return this.actions.pipe(
```
Comment (timdeschryver) [inline docs/rules/no-effect-decorator-and-creator.md:31]:
```suggestion
    return this.actions$.pipe(
```

**Review by timdeschryver — APPROVED** [review]

**Pull request merged**